mama-prd/README.md

```diff
@@ -1,12 +1,14 @@
 # mama-stage-based-reminders
-MAMA Stage Based Reminders Javascript sandbox application for production.
+MAMA Javascript sandbox application for production.
 
 Conversation configuration:
 
 * [Javascript](https://github.com/praekelt/mama-sms/)
 * [Configuration](config.json)
+* [Reports](reports.json)
 
 Raw URLs:
 
 * Javascript (go-mama-sms.js): [Raw](https://raw.githubusercontent.com/praekelt/mama-sms/develop/lib/go-mama-sms.js)
```

Comment:
I think this is because this is the 'old' version (you'll notice that URL doesn't work anymore, it's now at https://github.com/praekelt/mama-sms/blob/develop/old/lib/go-mama-sms.js).

It's still around because the new stuff uses the new metrics and our own dashboarding, the old version used Holodeck and MAMA SA is still using that code.